skbio/io/format/embl.py

```diff
@@ -0,0 +1,1497 @@
+"""
+EMBL format (:mod:`skbio.io.format.embl`)
+=========================================
+
+.. currentmodule:: skbio.io.format.embl
+
+EMBL format stores sequence and its annotation together. The start of the
+annotation section is marked by a line beginning with the word "ID". The start
+of sequence section is marked by a line beginning with the word "SQ" and the
+end of the section is marked by a line with only "//". More information on
+EMBL file format can be found here [1]_.
+
+The EMBL file may ends with .embl or .txt extension. An example of EMBL file
+can be seen here [2]_.
+
+Feature level products
+^^^^^^^^^^^^^^^^^^^^^^
+
+Feature-level products contain nucleotide sequence and related annotation
+derived from submitted ENA assembled and annotated sequences. Data are
+distributed in flatfile format, similar to that of parent ENA records,
+with each flatfile representing a single feature. While only the sequence
+of the feature is included in such entries, features are derived from the
+parent entry, and can't be applied as interval metadata. For such reason,
+interval metatdata are ignored from Feature-level products, as they will be
+ignored by subsetting a generic Sequence object. More information on
+Feature-level product can be found here [3]_.
+
+Format Support
+--------------
+**Has Sniffer: Yes**
+
+**NOTE: No protein support at the moment**
```

Comment:
That would be great, thank you. The intent is that, if a user wants to know more about the issue, they can track it down easily. And any development conversation surrounding this support has a home.